benchexec/tools/cmaesfuzz.py

```diff
@@ -29,6 +29,16 @@ def executable(self, tool_locator):
         return tool_locator.find_executable("fuzzer")
 
     def cmdline(self, executable, options, task, rlimits):
+        # add a time limit if not given
+        # that is hopefully sufficient to write all tests
+        if "-t" not in options:
```

Comment:
```suggestion
        if "-t" not in options and rlimits.cputime:
```
A time limit does not always exist.